### DISCUSSION TITLE
Include patch for improving plain text rendering

### DISCUSSION THREAD

**Discussion by RangerMauve** [bromite/bromite#2114] [Ideas]
Hey folks, I recently added a patch to make rendering of plain text files on mobile nicer since the default was to have them zoom out.

Would y'all be interested in a PR that adds the patch to Bromite as well?

https://github.com/AgregoreWeb/agregore-mobile/blob/default/patches/0001-AG-Improve-plain-text-rendering-on-mobile.patch

**Comment by csagan5** [comment]
Thanks for the heads-up; do you have some screenshot to see how it looks like?

**Reply by RangerMauve** [reply]
It just renders the text at a regular font size rather than zoomed out: 
![Screenshot_20220608-141251_Agregore](https://user-images.githubusercontent.com/911495/172688097-a3cb153e-fadc-4926-9e32-6d96cd0ef98a.jpg)

**Reply by csagan5** [reply]
I will consider it for inclusion on next release, thanks!

**Reply by csagan5** [reply]
It is now in `102.0.5005.96`: https://github.com/bromite/bromite/blob/102.0.5005.96/build/patches/Improve-plain-text-rendering-on-mobile.patch
